UseCases/SupplyChain/README.md

```diff
@@ -0,0 +1,41 @@
+# Broader Supply Chain Value Propositions and Use Cases
+To fully appreciate the value of the OSIM use cases 
+requires understanding of the broader supply chain uses
+in which OSIM plays a role.
+For example to see the value of information modeling to 
+software bill of materials (SBOM), 
+you first have to undershand the value of SBOMs.
+
+The use cases in this directory are broader than OSIM 
+and show the value of supply chain transparence in general.
+
+The purpose of these use cases is to demonstrate the benefits of 
+Software Bills of Materials (SBOMs) to software Producers and Consumers. 
+It strives to answer the questions: 
```

Comment:
Question: could we phrase the questions using `we` and `our` instead of `I` and `my`?
Rationale: 
>to me that reads more engaging and realistic for our topic (as organizations tend to prefer the collective perspective).

Reply:
added to PR76